### DISCUSSION TITLE
The format of variable supports pipeline.

### DISCUSSION THREAD

**Discussion by MicroOps-cn** [grafana/grafana#51685] [General]
I want to add a link to explore in one of my panels. The link will contain query statements generated by variables. The variables in the query statements need to be formatted with doublequote or regex,

Explore needs to URLEncode the query statement.

In other words, if I inject the variables of my market into the URL of explore, I need to format doublequote or regex first, and then URLEncode.

But I haven't found any way to realize this requirement. I think if the variable supports pipeline for multiple processing, this requirement should be realized.

For example: `${keywords:doublequote|percentencode}`

**Comment by MicroOps-cn** [comment]
I have implemented it on my own branch and hope to merge it into the main branch
#51859

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
